apps/whale-api/src/module.api/poolpair.controller.ts

```diff
@@ -203,6 +203,7 @@ function mapPoolPair (
   volume?: PoolPairData['volume']
 ): PoolPairData {
   const [symbolA, symbolB] = info.symbol.split('-')
+  const [nameA, nameB] = info.name.split('-')
```

Comment:
For PoolPair, this isn't a safe way to get `name` since the name of the underlying token can be updated without updating the poolpair. The `poolpair.name` can also be created without `-`, making this code fatal. 

I would recommend not implementing `name` for this code section if you don't need it.

Reply:
Oh I see, let me check. But I might need your help with this 😅

Reply:
Can I ask @fuxingloh, when will there be no `-`? I thought `name` on this part of the code is always a combination of tokenA and tokenB's names

Reply:
You can update a PoolPair token to any name after the creation of the token https://jellyfishsdk.com/node/jellyfish/api/token#updatetoken

Reply:
If you need the name, you can use `DeFiDCache` and use the token id to resolve the name instead. Since it's a cache the complexity is managed and won't be a performance issue.

https://github.com/JellyfishSDK/jellyfish/blob/0200dc74d77f6d331e531cad0db6e991f477f26a/apps/whale-api/src/module.api/poolpair.controller.ts#L28

Reply:
Ok I see. I will try to use DeFiDCache. Thank you!

Reply:
Updated. Let me know if I can improve anything. Thanks! 🙂